Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM registry.hub.docker.com/library/debian:stable-slim
 
-LABEL Maintainer="r.siudak@ultimaker.com" \
+LABEL Maintainer="software-embedded-platform@ultimaker.com" \
       Comment="Ultimaker python3-inotify environment"
 
 RUN apt-get update && \
-    apt-get install -y \
-    python3-stdeb \
-    python-all
+    apt-get install -q -y --no-install-recommends \
+        python3-all \
+        python3-stdeb && \
```

Comment:
to ease with copy/pasting, sorting and the sort, put the && \ on a new line. that way, you can do simply 'select packages, sort'

Reply:
Tried this, docker is unable to parse the line then because is does not recognize &&.